.vsts-ci.yml

```diff
@@ -11,8 +11,8 @@ queue:
 
 variables:
   BuildPlatform: 'Any CPU'
-  InsertTargetBranchFullName: 'lab/d16.0stg472'
-  InsertTargetBranchShortName: 'd16.0stg472'
+  InsertTargetBranchFullName: 'rel/d16.0'
+  InsertTargetBranchShortName: 'd16.0'
```

Comment:
@jmarolf What's the impact of this flowing like this? If this is bad, we need to find some not-in-the-repo way to configure this on a per-branch basis.